FAQ.md

```diff
@@ -0,0 +1,42 @@
+# FAQ
+
+- **[Networking and service meshes](#networking-and-service-meshes)**
+- **[Actors](#actors)**
+- **[Developer language SDKs and frameworks](#developer-language-sdks-and-frameworks)**
+
+## Networking and service meshes
+### How does Dapr work with service meshes?
+
+Dapr is a distributed application runtime.  Unlike a service mesh which is focused on networking concerns, Dapr is focused on providing building blocks that make it easier for developers to build microservices.  Dapr is developer-centric versus service meshes being infrastructure-centric.
+
+Dapr can be used alongside any service mesh such as Istio and Linkerd. A service mesh is a dedicated network infrastructure layer designed to connect services to one another and provide insightful telemetry. A service mesh doesn’t introduce new functionality to an application. 
+
+That is where Dapr comes in. Dapr is a language agnostic programming model built on http and gRPC that provides distributed system building blocks via open APIs for asynchronous pub-sub, stateful services, service discovery and invocation, actors and distributed tracing. Dapr introduces new functionality to an app’s runtime. Both service meshes and Dapr run as side-car services to your application, one giving network features and the other distributed application capabilities. 
+
+### How does Dapr interoperate with the service mesh interface (SMI)? 
+SMI is an abstraction layer that provides a common API surface across different service mesh technology.  Dapr can leverage any service mesh technology including SMI.
+
+### What’s the difference between Dapr and Istio? 
+Read [How does Dapr work with service meshes?](https://github.com/dapr/dapr/wiki/FAQ#how-does-dapr-work-with-service-meshes) Istio is an open source service mesh implementation that focuses on Layer7 routing, traffic flow management and mTLS authentication between services. Istio uses a sidecar to intercept traffic going into and out of a container and enforces a set of network policies on them. 
+
+Istio is not a programming model and does not focus on application level features such as state management, pub-sub, bindings etc. That is where Dapr comes in. 
+
+## Actors
+### How does Dapr relate to Orleans and Service Fabric Reliable Actors?
+The actors in Dapr are based on the same virtual actor concept that [Orleans](https://www.microsoft.com/research/project/orleans-virtual-actors/) started, meaning that they are activated when called and garbage collected after a period of time. If you are familiar with Orleans, Dapr C# actors will be familiar. Dapr C# actors are based on [Service Fabric Reliable Actors](https://docs.microsoft.com/azure/service-fabric/service-fabric-reliable-actors-introduction) (which also came from Orleans) and enable you to take Reliable Actors in Service Fabric and migrate them to other hosting platforms such as Service Fabric Mesh, Kubernetes or other on-premise environments.

```

Comment:
Can we remove Service Fabric Mesh from here as the name of fully managed serverless service is not clear yet.

Reply:
Yes, remove Service Fabric Mesh, since this name will be deprecated.